lib/ui/screens/login.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:zachranobed/constants.dart';
+import 'package:zachranobed/ui/widgets/button.dart';
+import 'package:zachranobed/ui/widgets/clickableText.dart';
+import 'package:zachranobed/ui/widgets/textField.dart';
+
+class Login extends StatefulWidget {

```

Comment:
Login by mel byt spise StatelessWidget (sam si Uzivatele neuklada, bere si ho - a pushuje ho do Providera, ktery je uplne navrchu stromu.

Na tehle strance by mel tedy byt po kliknuti na Login button `Provider.of<User>(context, listen: false).newUser(xyz)`, ktery zaloguje noveho uzivatele a pushne se na home stranku. Do budoucna tu bude i API call (a tedy Repository pattern), ale to ted neni treba resit.

https://docs.flutter.dev/development/data-and-backend/state-mgmt/simple#providerof